spring-data-rest-webmvc/src/main/java/org/springframework/data/rest/webmvc/RepositoryRestController.java

```diff
@@ -46,4 +47,9 @@
 @Target({ ElementType.TYPE, ElementType.ANNOTATION_TYPE })
 @BasePathAwareController
 public @interface RepositoryRestController {
+    @AliasFor("path")
+    String value() default "";
+
+    @AliasFor("value")
+    String path() default "";
```

Comment:
@RequestMapping allows specifying more than one path, should we do the same here?

Reply:
@alienisty I agree, I update it